electrumsv/wallet_support/late_header_worker.py

```diff
@@ -103,9 +103,9 @@ async def late_header_worker_async(wallet_data: WalletDataAccess, state: LateHea
                     hash_to_hex_str(tsc_proof.transaction_hash))
                 state.block_transactions[tsc_proof.block_hash].add(tsc_proof.transaction_hash)
             else:
-                await _process_one_merkle_proof(wallet_data, state, tsc_proof.transaction_hash,
-                    TxFlags.STATE_CLEARED, tsc_proof.block_hash, tsc_proof.to_bytes(), header)
-        elif item_kind == PendingHeaderWorkKind.NEW_HEADER:
+                await _process_one_merkle_proof(wallet_data, state, tsc_proof.block_hash,
```

Comment:
Why have you switched block hash and transaction hash when the function takes the arguments in the existing order before this change?

Reply:
Rebasing error...